__tests__/patch-article-by-id-api.test.js

```diff
@@ -0,0 +1,79 @@
+const request = require("supertest");
+
+const db = require("../db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("404: responds with an error msg if article does not exist", () => {
+    return request(app)
+      .patch("/api/articles/999")
+      .send({ inc_votes: 1 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+  test("404: responds with an error msg if invalid inc_votes value", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "invalid" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("201: responds with a updated article with a given article id", () => {

```

Comment:
Have we created something or have we updated something? 201 is for new created content. Here a 200 would be more appropriate.